frontend/src/metabase/lib/redux/utils-ts.ts

```diff
@@ -0,0 +1,16 @@
+import { createAction as createActionOriginal } from "redux-actions";
+
+import type { Action, ActionFunctionAny } from "redux-actions";
+
+export { handleActions } from "redux-actions";
+
+export function createAction(
+  actionType: string,
+): ActionFunctionAny<Action<any>>;
+export function createAction<PayloadCreator>(
+  actionType: string,
+  payloadCreator: PayloadCreator,
+): PayloadCreator;
+export function createAction(actionType: any, payloadCreator?: any) {
```

Comment:
This is necessary otherwise calling `dismissUndo`, which is created via `createAction`, in TypeScript files would result in the following error.
<img width="601" alt="image" src="https://user-images.githubusercontent.com/1937582/235946313-1d49a6ff-3810-4fd2-b0b6-36af1351cea8.png">